src/runtime/adaptivity_engine.cpp

```diff
@@ -153,7 +153,17 @@ bool is_likely_invariant_argument(common::db::kernel_entry &kernel_entry,
 }
 
 int determine_ptr_alignment(uint64_t ptrval) {
-#if __has_builtin(__builtin_ctz)
+#if defined(__GNUC__) 
```

Comment:
Did Clang always support `__builtin_clz`?
This only tests that the compiler has some GNU C extensions: https://stackoverflow.com/a/55926503

Reply:
Ah damn... clang at least has `__has_builtin` which is certainly supported for all clang >= 14 which we support

Reply:
Okay, I think it should work now correctly :)